src/common/components/dropdown/service-provider-dropdown/ServiceProviderDropdown.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <p-select-dropdown class="service-provider-dropdown"
+                       :selected="proxySelected"
+                       :items="contextMenuItems"
+                       @select="handleSelect"
+    >
+        <span v-if="Object.keys(providerList).length > 0" class="text">
```

Comment:
```suggestion
        <span v-if="providerList[proxySelected]" class="text">
```

이렇게 바꾸는 것을 제안합니다!

이유는 `providerList`를 array로 변환하는 과정 없이 해결할 수 있으며, 아래에서 `providerList[proxySelected]`를 사용하므로 "`providerList[proxySelected]`가 있는 경우에만 보여주겠다"는 의미가 명확히 보이도록하면 가독성이 훨씬 높아지는 것 같습니다.

Reply:
훨씬 보기좋네여👍 👍 👍